src/OpenTelemetry.Instrumentation.Wcf/OpenTelemetry.Instrumentation.Wcf.csproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="System.ServiceModel.Primitives" Version="4.7.0" />
+    <PackageReference Include="System.ServiceModel.Http" Version="4.7.0" />
```

Comment:
Mixed feelings about adding this reference. Not sure if anyone will complain. Seems like we only need `HttpRequestMessageProperty` what if we accessed it reflectively and only performed the http actions if we detect the class/assembly is there?

If we do keep the ref, it looks like `System.ServiceModel.Http` pulls in `System.ServiceModel.Primitives` so you could probably remove that explicit reference.

Semi-related, how do you feel about updating the README to explicitly say we have tested this package with SOAP + HTTP transports and they will work but others may not?

Reply:
I like that idea, no need to bring in the reference if the application isn't using HTTP.  Done 👍